UITests/README.md

```diff
@@ -0,0 +1,44 @@
+# UWPControls
+
+UWPControls is a collection of test scenarios that covers basic interactions with [Universal Windows Platform controls (UI elements)](https://docs.microsoft.com/en-us/windows/uwp/controls-and-patterns/). Use this as a reference on how to interact with certain UWP controls you have in your application.
```

Comment:
I don't understand this paragraph. What is "UWPControls"? Does that refer to the Xaml Controls Gallery? I don't think we need this paragraph if so.